sections/futures/MobileTrade/UserTabs/UserTabs.tsx

```diff
@@ -29,17 +31,19 @@ const TABS = [
 	{
 		title: 'Transfers',
 		component: <TransfersTab />,
+		type: 'isolated_margin',
 	},
 ];
 
 const UserTabs: React.FC = () => {
 	const [activeTab, setActiveTab] = React.useState(0);
+	const accountType = useAppSelector(selectFuturesType);
 
 	return (
 		<Container>
 			<UserTabsContainer>
 				<TabButtonsContainer>
-					{TABS.map(({ title }, i) => (
+					{TABS.filter(({ type }) => !type || type === accountType).map(({ title }, i) => (
```

Comment:
Conditional orders should also be filtered out to not show on isolate margin actually

Reply:
Will make the changes.